apps/wrapper/src/renderer/src/components/Fileclass.svelte

```diff
@@ -0,0 +1,155 @@
+<script lang="ts">
+  import logo from '../assets/logo(normal).png';
+  import { onMount } from 'svelte';
+  import { appState } from '../stores';
+
+  let fileClass = '';
+  let appName = '';
+  let proceed = false;
+  let checkLoading = false;
+
+  onMount(async () => {
+    await window.api.getFileClass('3be2565e-fe51-4e9d-820f-d6c1315d204d');
+    let classInterval = setInterval(async () => {
+      const fileClassResponse = await window.api.checkFileClass();
+      if (fileClassResponse.code === 200) {
+        fileClass = fileClassResponse.message;
+        clearInterval(classInterval);
+      }
+    }, 1000);
+    const appNameRes = await window.api.getAppName();
+    appName = appNameRes.message;
+  });
+
+  async function topSecretChecks() {
+    if (fileClass === 'TOPSECRET') {
+      checkLoading = true;
+      const response: { code: number; message: string } = await window.api.secretChecks();
+      proceed = true;
+      console.log(response.code);
+      if (response.code === 200) {
+        appState.set({
+          passedCheck: true,
+          currentState: 'fileClass',
+        });
+      } else {
+        proceed = false;
+        appState.set({
+          passedCheck: false,
+          currentState: 'fileClass',
+        });
+      }
+    } else if (fileClass === 'SENSITIVE') {
+      proceed = true;
+      appState.set({
+        passedCheck: true,
+        currentState: 'fileClass',
+      });
+    } else if (fileClass === 'OPEN') {
+      proceed = true;
+      appState.set({
+        passedCheck: true,
+        currentState: 'viewDoc',
+      });
+    }
+  }
+</script>
+
+<div class="hidden" class:hidden={proceed}>
+  <h1 class="text-center mt-20 -mb-24 text-3xl font-bold">Armadillo Verification</h1>
+
+  <div class="h-screen grid">
+    <div class="place-self-center">
+      <img src={logo} class="h-52 mb-4 m-auto" alt="logo" />
+
+      <h1 class="text-center text-lg"
+        >You are trying to access file:
+        <span class="text-secondary">
+          {#if appName === ''}
+            <span class="text-white loading loading-spinner loading-md mx-2 -mb-2"></span>
+          {:else}
+            {appName}
+          {/if}
+        </span>
+      </h1>
+      <h1 class="font-bold text-center text-lg"
+        >File Classification:
```

Comment:
If prettier suggests to move the `>` to a new line, make the `>` on the same line as the element, and it's children in a new line.

```suggestion
      <h1 class="font-bold text-center text-lg">
        File Classification:
```